examples/custom/build.sh

```diff
@@ -2,7 +2,9 @@
 set -e
 
 if ! [ -x "$(command -v ko)" ]; then
-    GO111MODULE=on go get -mod=readonly github.com/google/ko/cmd/ko
```

Comment:
making change in examples. is this intentional?
Shd you make this in integration?